static/app/views/dashboards/widgets/common/widgetFrame.tsx

```diff
@@ -26,6 +31,30 @@ export function WidgetFrame(props: Props) {
           {description && showDescriptionInTooltip && (
             <QuestionTooltip size="sm" title={description} />
           )}
+
+          {actions && (
```

Comment:
super minor nit, should we care about checking if `actions.length > 0` so we avoid rendering an empty div?

Reply:
🤔 yes definitely! Forgot for a second that `Boolean([])` is `true` 👍🏻